cfgov/unprocessed/css/molecules/global-eyebrow.less

```diff
@@ -111,6 +111,7 @@
     padding-right: unit(@grid_gutter-width / 2 / @base-font-size-px, rem);
     padding-bottom: unit(@grid_gutter-width / @base-font-size-px, rem);
 
+    // Only applied if the tagline was included in the global eyebrow template.
     .a-tagline {
       display: none;
```

Comment:
Are these lines still needed, if we're not even rendering the tagline HTML now?

Reply:
Yea I waffled on this, however it is still correct in respect to the API. You could create `{{ global_eyebrow.render() }}` and the tag line would be included, but should be hidden in the list-style layout, in which case the CSS should hide it. We don't do that in the template and maybe never will, but the signature for `global_eyebrow.render` allows it, so I opted to leave the CSS since it's just one rule. I can be convinced otherwise.